### ISSUE TITLE
How apply patchs on Xenomai

### ISSUE THREAD

**Issue by sestrem** [nolange/cmake_xenomai#1]
Hello Norberto,

First of all, congratulations for this project and a big thank you for sharing wit the community.

Could you please help me with the step [Patching Xenomai](https://github.com/nolange/cmake_xenomai)?

I'm running Xenomai in a Raspbian [installed following these steps](https://lemariva.com/blog/2018/07/raspberry-pi-xenomai-patching-tutorial-for-kernel-4-14-y). 

I can run and get results with `sudo /usr/xenomai/bin/latency` and `sudo /usr/xenomai/bin/xeno-test` but I don't know how apply the patchs on it.

Thanks in advance,
Evandro Sestrem

**Comment by nolange** [comment]
Did you try following the method in https://www.xenomai.org/pipermail/xenomai/2018-December/040066.html ?

You need to apply the patchset on the extracted xenomai-3.0.7.tar.bz2 sources. cd to that directory and run
```bash
for ipatch in PATH_TO/xeno_cmake/patchset/*; do patch -p1 < "$ipatch"; done
```

**Comment by sestrem** [comment]
When I applied the patches:
https://i.postimg.cc/TYyPnh7m/imagem1.png

But when I tried to compile the https://github.com/nolange/cmake_xenomai/tree/master/examples/template I got:
https://i.postimg.cc/2jWk5gVF/imagem2.png

Do I need to execute the ""Installing the Cobalt core and "Installing the Mercury core" steps from https://gitlab.denx.de/Xenomai/xenomai/wikis/Installing_Xenomai_3?

**Comment by nolange** [comment]
No, you need to follow the instructions from [Installing the Xenomai libraries and tools](https://gitlab.denx.de/Xenomai/xenomai/wikis/Installing_Xenomai_3#library-install).

The patchset is an attempt to get this upstream, and thus automatically installed, the downside is that this only happens if you build from source.
You could install the cmake files yourself with the `config/install_cmakeconfig.sh` script (see sections `Config-files` and Bootstrap header` example is in the README).

PS. please don't make screenshots in the future, just copy the text.
